pyproject.toml

```diff
@@ -122,7 +122,7 @@ exclude = [
     'render/renderer/v3/suite_profile_notebook_renderer\.py',  # 4
     'render/util\.py',  # 2
     'render/view/view\.py',  # 11
-    'rule_based_profiler/attributed_resolved_metrics\.py',  # 4
+    'rule_based_profiler/attributed_resolved_metrics\.py',  # 1
```

Comment:
Can you delete this line?
Even if it means ignoring the 1 remaining type error it's worth it to get this under coverage.

Reply:
@Kilo59 Done -- thanks.